Core/ALife.Core/Utility/Colours/ColourExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace ALife.Core.Utility.Colours
+{
+    /// <summary>
+    /// Various extension methods for the IColour interface.
+    /// </summary>
+    public static class ColourExtensions
+    {
+        /// <summary>
+        /// Converts the current IColour object to a Colour.
+        /// </summary>
+        /// <param name="colour">The colour.</param>
+        /// <returns>The Colour object.</returns>
+        public static Colour ToColour(this IColour colour)
+        {
+            return new Colour(colour.A, colour.R, colour.G, colour.B, colour.WasPredefined);
+        }
+
```

Comment:
What's the point of this, exactly. What "IColours" aren't' "Colours"

Reply:
I've got HSL and HSV colours defined as well (that I ended up separating from the main Colour class as I'd shown you previously).